SmartDeviceLink-iOS.xcodeproj/project.pbxproj

```diff
@@ -4925,7 +4932,8 @@
 				5D61FD0C1A84238C00846EE7 /* SDLOnEncodedSyncPData.m in Sources */,
 				5D61FD0A1A84238C00846EE7 /* SDLOnDriverDistraction.m in Sources */,
 				5D61FCED1A84238C00846EE7 /* SDLListFiles.m in Sources */,
-				DA9F7E801DCC028B00ACAE48 /* SDLOnWayPointChange.m in Sources */,
+				DA9F7E801DCC028B00ACAE48 /* SDLOnWaypointChange.m in Sources */,
```

Comment:
This changes a file name here to the wrong capitalization

Reply:
Oh... that was a mistake resolving a conflict...